cloud_chatops/.pylintrc

```diff
@@ -4,5 +4,7 @@
 max-line-length=120
 
 # Disable various warnings:
+# E0401 Ignored as imports work and will be tested when deployed in dev
```

Comment:
This needs to be fixed, it indicates a problem with the code structure
It's especially useful when we upgrade Python major versions and libraries end up in the Python std lib